### DISCUSSION TITLE
Default Value on controller continues to say the type is undefined and nothing is assignable

### DISCUSSION THREAD

**Discussion by hines-sharrod** [react-hook-form/react-hook-form#9199] [Q&A]
![image](https://user-images.githubusercontent.com/8672599/195144456-9cfda189-c5d7-4ecc-8a7c-b02108f9e72f.png)

![image](https://user-images.githubusercontent.com/8672599/195144049-eccffda0-230e-40c7-beee-1a16f6850873.png)

Passing in a string or number keeps getting the same error above. The app only functions once adding string and number as unions to the controller.d.ts.file that is throwing the error (below).

![image](https://user-images.githubusercontent.com/8672599/195144308-b3a2feab-54b7-4541-8d91-ba61c53bd367.png)

Any thoughts on how I can fix this?

**Comment by bluebill1049** [comment]
consider to share a codesandbox and probably easier to provider a defaultValues at useForm hook
